core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -271,7 +331,24 @@ case class MultiClassificationMetrics
   Recall: Double,
   F1: Double,
   Error: Double,
-  ThresholdMetrics: MulticlassThresholdMetrics
+  ThresholdMetrics: MulticlassThresholdMetrics,
+  TopKMetrics: MultiClassificationMetricsTopK
+) extends EvaluationMetrics
+
+/**
+ * Metrics for topK multiclass classification
+ *
+ * @param PrecisionTopK
+ * @param RecallTopK
+ * @param F1TopK
+ * @param ErrorTopK
+ */
+case class MultiClassificationMetricsTopK
+(
+  PrecisionTopK: Seq[Double],
```

Comment:
the class already has TopK in the name, no need to also include it each of the member names.

Reply:
Agreed, we also need to store the topK values themselves so it's clear what the metric sequences are coming from